assembl/static2/tests/unit/components/voteSession/__snapshots__/Proposal.spec.jsx.snap

```diff
@@ -862,11 +862,7 @@ exports[`Proposal component should match Proposal snapshot 1`] = `
         </p>
         <div
           className="rc-slider"
-          onBlur={[Function]}
```

Comment:
Woops this commit looks like it breaks tests actually. Maybe not include it in this PR?

Reply:
Sigh. It repaired tests for me; that's its only raison d'être. Feel free to ``it reset --hard HEAD~1; git push --force``.

Reply:
We found some difference in the past with the version of node you use to execute the tests. Be sure to execute the tests with the virtualenv enabled so we use the same node version.